Classes/LoginProvider/ShibbolethLoginProvider.php

```diff
@@ -45,7 +45,7 @@ public function render(StandaloneView $view, PageRenderer $pageRenderer, LoginCo
         }
         $view->setTemplatePathAndFilename(GeneralUtility::getFileAbsFileName($template));
 
-        $target = GeneralUtility::getIndpEnv('TYPO3_REQUEST_HOST') . '/typo3';
+        $target = GeneralUtility::getIndpEnv('TYPO3_REQUEST_HOST') . '/typo3/?login_status=login';
         if ($extensionConfiguration['forceSSL'] && !GeneralUtility::getIndpEnv('TYPO3_SSL')) {
```

Comment:
@lorenzulrich I'm not sure from our last chat whether you suggested that we should not add the parameter in the PHP code. If we don't add it there, where and how would you suggest we include the "login_status=login" parameter?

Reply:
When we discussed this, I wasn't aware that the URI is generated in PHP code (I thought it was configuration). Therefore, placing it here is OK.